modules/services/openssh.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, ... }:
+
+let
+  cfg = config.services.openssh;
+in
+{
+  options = {
+    services.openssh.enable = lib.mkOption {
+      type = lib.types.nullOr lib.types.bool;
+      default = null;
+      description = ''
+        Whether to enable an OpenSSH server.
+
+        The default is null which means let macOS manage the OpenSSH server.
+      '';
+    };
+
+    services.openssh.useAppleServer = lib.mkOption {
+      type = lib.types.bool;
+      default = true;
+      example = false;
+      description = ''
+        Whether to use Apple's built-in OpenSSH server.
+      '';
+    };
+  };
```

Comment:
Honestly I think we can just drop this if it defaults to `true`, since nobody is ever going to set it to `false` until/unless it’s implemented, and I guess time has mellowed me into thinking it might be too annoying to force people to explicitly toggle it to use this :)